test/why.js

```diff
@@ -883,6 +884,62 @@ test('bigints', { skip: !hasBigInts }, function (t) {
 	t.end();
 });
 
+test('toPrimitive', { skip: !toPrimitive }, function (t) {
```

Comment:
```suggestion
test('toPrimitive', function (t) {
```
this can't ever be falsy